collections/src/main/scala/strawman/collection/package.scala

```diff
@@ -1,15 +1,45 @@
 package strawman
 
-import scala.{Any, AnyVal, Array, Boolean, Char, IllegalArgumentException, IndexOutOfBoundsException, Int, NoSuchElementException, Unit, UnsupportedOperationException, PartialFunction, Option, None, Some}
+import scala.{Any, AnyVal, Array, Boolean, Char, IllegalArgumentException, IndexOutOfBoundsException, Int, NoSuchElementException, Unit, UnsupportedOperationException, PartialFunction, Option, None, Some, deprecated}
 import scala.Predef.{String, ArrowAssoc}
 import scala.reflect.ClassTag
 
 package object collection extends LowPriority {
+  @deprecated("Use Iterable instead of Traversable", "2.13.0")
+  val Traversable = Iterable
+  @deprecated("Use SeqOps instead of SeqLike", "2.13.0")
+  type SeqLike[A, T] = SeqOps[A, Seq, T]
+
+  @deprecated("Gen* collection types have been removed", "2.13.0")
+  type GenTraversableOnce[+X] = IterableOnce[X]
+  @deprecated("Gen* collection types have been removed", "2.13.0")
+  val GenTraversableOnce = IterableOnce
+  @deprecated("Gen* collection types have been removed", "2.13.0")
+  type GenTraversable[+X] = Iterable[X]
+  @deprecated("Gen* collection types have been removed", "2.13.0")
+  val GenTraversable = Iterable
+  @deprecated("Gen* collection types have been removed", "2.13.0")
+  type GenIterable[+X] = Iterable[X]
+  @deprecated("Gen* collection types have been removed", "2.13.0")
+  val GenIterable = Iterable
+  @deprecated("Gen* collection types have been removed", "2.13.0")
+  type GenSeq[+X] = Seq[X]
+  @deprecated("Gen* collection types have been removed", "2.13.0")
+  val GenSeq = Seq
+  @deprecated("Gen* collection types have been removed", "2.13.0")
+  type GenSet[X] = Set[X]
+  @deprecated("Gen* collection types have been removed", "2.13.0")
+  val GenSet = Set
+  @deprecated("Gen* collection types have been removed", "2.13.0")
+  type GenMap[K, +V] = Map[K, V]
+  @deprecated("Gen* collection types have been removed", "2.13.0")
+  val GenMap = Map
```

Comment:
If we define the `GenXxx` types here then it will not be possible to define them in the parallel collections module, as proposed in #328. That being said, several people have expressed arguments *against* having such generic collection types anyway, so maybe that’s fine…

Reply:
We can remove them from here if we add them to the parallel collections. In that case you would need to parallel collections module to compile old code that uses these types.

Reply:
Let’s keep these aliases here for now, when we will work on the parallel collections we will move them.